management-api-agent-dse-6.8/pom.xml

```diff
@@ -30,7 +30,7 @@
     </repository>
   </repositories>
   <properties>
-    <dse.version>6.8.44</dse.version>
+    <dse.version>6.8.45</dse.version>
```

Comment:
Should this be 6.8.46?

Reply:
good catch, I thought I might have missed one somewhere